client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.ts

```diff
@@ -287,9 +240,6 @@ export function getEnhancedTasks(
 					break;
 				case 'link_in_bio_launched':
 					taskData = {
-						title: translate( 'Launch your site' ),
-						completed: siteLaunchCompleted,
-						disabled: ! linkInBioLinksEditCompleted,
 						isLaunchTask: true,
```

Comment:
I wonder if we should be defining the isLaunchTask attribute on the backend as well?

Reply:
This is complicated and requires a backend change. We should defer to a follow up.